Makefile

```diff
@@ -12,7 +12,7 @@ core2smtlib2_unsupported_ops = "while" "!=" "exp" "exp2" "expm1" "log" "log10" "
 core2smtlib2_unsupported_consts = "LOG2E" "LOG10E" "M_1_PI" "M_2_PI" "M_2_SQRTPI"
 
 core2sollya_prec = binary32 binary64
-core2sollya_unsupported_ops = "isinf" "isnan" "isfinite" "isnormal" "signbit" "fabs" "fma" "exp2" "cbrt" "hypot" "atan2" "tgamma" "lgamma" "fmod" "remainder" "fmax" "fmin" "fdim" "copysign" "trunc" "round" "nearbyint"
+core2sollya_unsupported_ops = "isnormal" "tgamma" "lgamma" "remainder" "fmod" "round" "cbrt" "atan2"
```

Comment:
Brief recap of status of ops:
  - tgamma and lgamma would have to be reimplemented from scratch; sollya doesn't seem to have gamma functions
  - remainder and fmod are hard, but could be implemented with exact division or sollya's mod function
  - isnormal doesn't have a sollya implementation, but could be implemented by table lookup for common precisions
  - round is a little tricky, as there doesn't seem to be an easy way to get the desired "round nearest away" behavior
  - cbrt and atan2 have obvious implementations, but they don't quite work as expected, because sollya is too smart for the ieee 754 standard

Note also that a few other functions (signbit, copysign, isinf/isnan/isfinite) will have nonstandard behavior, because sollya doesn't believe in negative zero, and has its own ideas about infinite and nan values.

Reply:
This seems pretty acceptable. It's unfortunate that there are weird quirks and gotchas, but that's the way it is with FP. I would say that not supporting, say, remainder, round, and fmod seems pretty OK to me.